client/app/assets/less/redash/redash-newstyle.less

```diff
@@ -60,23 +60,23 @@ body {
 
 // Fixed width layout for specific pages
 @media (min-width: 768px) {
-  settings-screen, home-page, page-dashboard-list, page-queries-list, alerts-list-page, alert-page, queries-search-results-page, .fixed-container {
+  settings-screen, home-page, page-dashboard-list, page-queries-list, page-alerts-list, alert-page, queries-search-results-page, .fixed-container {
```

Comment:
Out of the scope of this PR, but all this tag name based rules will no longer work once we move to React. Worth switching to a (single) class name soon.